### PR TITLE
Installed Shadcn/UI and Tailwind. Created new file for interfaces

### PR DESCRIPTION
**Pull request by Allen-Craig**
Completed install on Shadcn UI and Tailwind. I also created a file for the interface in case there are more in the future.

### CODE REVIEW

File: postcss.config.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export default {
```
Comment (Bill-Compton) [inline postcss.config.js]: What is the use of this file. It doesn't appear to have any configs, just empty objects?

File: src/interfaces.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+export interface TeamData {
```
Comment (Bill-Compton) [inline src/interfaces.tsx]: I think we could set this up as a type

`export type TeamData = {
  SeasonType: number;
  Season: number;
  Conference: string;
  Division: string;
  Team: string;
  Name: string;
  Wins: number;
  Losses: number;
  Ties: number;
  NetPoints: number;
  Touchdowns: number;
  DivisionWins: number;
  TeamID: number;
}`

File: tailwind.config.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    darkMode: ["class"],
+    content: [
+    "./pages/**/*.{js,ts,jsx,tsx}",
+    "./components/**/*.{js,ts,jsx,tsx}",
+    "./src/**/*.{js,ts,jsx,tsx}",
+  ],
+  theme: {
+  	extend: {
+  		borderRadius: {
+  			lg: 'var(--radius)',
```
Comment (Bill-Compton) [inline tailwind.config.js]: Curious why you added the borderRadius theming?

**Review by Bill-Compton — APPROVED** [review]
I think this looks good so far. I will read more into the docs for the tools as I haven't messed with them a lot. Can we also move this to a more standard port like 3000 or something?

File: postcss.config.js (hunk 1 above)
Comment (Allen-Craig) [reply postcss.config.js]: I'll remove it. I added it when installing Shadcn AND Tailwind but never used it.

File: tailwind.config.js (hunk 3 above)
Comment (Allen-Craig) [reply tailwind.config.js]: Just playing around, no particular reason. I removed it.

Comment (Allen-Craig) [reply tailwind.config.js]: I have been watching some videos on tailwind. I had no idea what it was until you mentioned it.

File: src/interfaces.tsx (hunk 2 above)
Comment (Allen-Craig) [reply src/interfaces.tsx]: I have changed it to a type. How do you decide to use a type or interface?
Is it strictly based on inheritance??

**Pull request merged**